### PR TITLE
Draft/RFC: Slight rework

### PR DESCRIPTION
**Pull request by laenas**
- Just hijacked the fsdocs default styling, but would probably want to get it cleaner based on the snippets we end up wanting to use.
- Also just borrowed a reasonably verbose sample from FSharp.Data as a fancier ipsum-lorem.
- Ideally, it occurs to me, hooking some highlighting of our key messaging (either across the header and/or across the 3 main-point columns into the snippets could be nice?  Succinct -> here's a cleverly succinct snippet.  Robust?  The same.
- I am no graphic designer, and so getting some slightly more refined pipes between the start-here steps, probably as (background) images would make them more subtle than their current iteration.
- I tweaked the copy of those 3 points to *end* on an `F# (...)` note - my eyes were 'tuning out' those paragraphs because by starting with F#, brain went into memetic-defense mode.  By making a point, and *then* tying back in the name, it seemed to feel more tying things together and ending on the branding.
- Foundation stuff is vaguely place-holder-y, it captures the essence of things, but needs to be made a little more digestible, possibly folding in the sponsor roll into that 'section' instead.

In short:  this is introductory idea stuff, rather than merge-ready PR.  If we like it, let's refine it - if we don't, let's not dump more energy down this path :)

### CODE REVIEW

**Comment by dsyme** [comment]
There are definite improvements here which we can lift straight in - especially to the messaging in "Getting Started".  I won't pull it directly but will take that wording and iterate

**Comment by dsyme** [comment]
> There are definite improvements here which we can lift straight in - especially to the messaging in "Getting Started". I won't pull it directly but will take that wording and iterate

That is, I'll first take the things that are "really very obviously better" :) Then we can iterate and consider the rest.

Thank you so much for doing this!!!

**Comment by laenas** [comment]
Of course, if this general direction is a good start, I can keep chipping away down this path. :)

**Comment by dsyme** [comment]
Closing this old PR. @laenas thank you for taking the time to do this draft
